examples/kubernetes/static-provisioning/filesystem/pv.yaml

```diff
@@ -18,4 +18,4 @@ spec:
     volumeHandle: "fsvol-0281aee0779ea975e"
     volumeAttributes:
       dnsName: "fs-078106df77a01f1f7.fsx.us-east-1.aws.internal"
-      volumeType: "filesystem"
```

Comment:
why change volumeType to type? I prefer the clarity, but don't feel too strongly about it

Reply:
`VolumeType` is a field in our CreateVolume API. Since the code is now dynamic, the user can technically add that field. I have a check that errors if the user inputs a field the CSI driver populates, so I needed to change the name to avoid the conflict.